net/dnscrypt-proxy2/Makefile

```diff
@@ -60,10 +76,19 @@ define Package/dnscrypt-proxy2/description
   such as DNSCrypt v2 and DNS-over-HTTPS.
 endef
 
+define Build/Prepare
```

Comment:
I do not understand defining a Prepare section is there is nothing else than just the default